.storybook/manager-head.html

```diff
@@ -401,4 +401,9 @@
   .search-field {
     display: none !important;
   }
+
+  a[class*="sto-"][data-selected="true"]{
```

Comment:
I have concerns that this might create confusion between the selected item and an item with focus as this looks a bit like a focus ring.

We should likely have a forced color media query and use the appropriate system color tokens

Something like this:
```css
@media (forced-colors: active) {
  a[class*="sto-"][  data-selected="true"] {
    background-color: SelectedItem;
    color: SelectedItemText;
  }
}
```

Reference of the System Colors: https://drafts.csswg.org/css-color-4/#css-system-colors
Forced colors media query: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors

Reply:
This is a very good resource! I have updated as recommended. In terms of selected items and focus, I always used this _hack_ where I place a transparent border that automatically shows up in high contrast modes as required. But this makes more sense because now it's clearer what a selected item vis-a-vis a focused one looks like.